.github/ISSUE_TEMPLATE/change.yml

```
@@ -0,0 +1,99 @@
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Rust Project Developers
+
+name: Change
+description: File a change for the FLS
+title: "[Change]: "
+labels: ["change intake"]
+assignees:
+  - PLeVasseur
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing a change!
+  - type: input
+    id: title
+    attributes:
+      label: Change description
+      description: Description of what is intended to be changed
+      placeholder: Fix description of lexing due to xx
+    validations:
+      required: true
+  - type: dropdown
+    id: area
+    attributes:
+      label: Area
+      description: Which part of the language does this fall under?
+      options:
+        - ABI
+        - async / await
+        - attributes
+        - closures
+        - conditional compilation
+        - const evaluation
+        - conventions
+        - destructors
+        - diagnostics
+        - "edition: 2015"
+        - "edition: 2018"
+        - "edition: 2021"
+        - "edition: 2024"
+        - editions
+        - generics
+        - glossary
+        - grammar
+        - inline assembly
+        - linkage
+        - macros
+        - method call
+        - panic
+        - patterns
+        - proc macros
+        - resolution (names, paths, namespaces, preludes, ...)
+        - types
+        - type inference
+        - type layout
+        - undefined behavior
+        - N/A
+    validations:
+      required: true
+  - type: dropdown
+    id: category
+    attributes:
+      label: Category
+      description: What kind of change is this?
+      options:
+        - Bug
+        - Formatting
+        - Improvement to Non-Prose Section
+        - Language Cleanup
+        - New Content
+        - Other
```

```suggestion
        - Bug
        - Formatting
        - Improvement to non-prose section
        - Language cleanup
        - New content
        - Other
```

Sentence-casing seems perhaps better for this.